### PR TITLE
Collection converter

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

File: src/converters/collection-converter.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+		view: {first, last, next},
+		totalItems,
+		numberOfItems,
+		member,
+	} = thing.attributes;
+
+	let items = [];
+
+	if (member.length !== 0) {
+		items = member.map(item => conversionHandler.convert(item.thing));
```
Comment (javiergamarra) [inline src/converters/collection-converter.js]: I prefer doing affirmative ifs, that way you don't have to negate the conditional in your head

Comment (javiergamarra) [reply src/converters/collection-converter.js]: Also ternaries you could do just the map right?

File: src/converters/collection-converter.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	} else {
+		items = member;
+	}
+
+	return new Collection(
+		id,
```
Comment (javiergamarra) [inline src/converters/collection-converter.js]: hate sf

File: src/model/collection.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+		this.first = first;
+		this.last = last;
+		this.next = next;
+		this.items = items;
+		this.totalItems = totalItems;
+		this.numberOfItems = numberOfItems;
+	}
+
+	/**
+	 * Checks wether this collection has another page with more items
```
Comment (javiergamarra) [inline src/model/collection.js]: typo :P

File: src/model/collection.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+	 * @return {boolean}
```
Comment (javiergamarra) [inline src/model/collection.js]: @review

File: src/model/thing.js [hunk 5]
```diff
@@ -16,6 +16,15 @@ export default class Thing {
 		this.operations = operations;
 	}
 
+	/**
+	 * Check wether the thing is a concrete type or not.
```
Comment (javiergamarra) [inline src/model/thing.js]: typo and @review

**Review by javiergamarra — APPROVED** [review]

**Comment by victorg1991** [comment]
Fixed :D

**Pull request merged**